test/airline/AirlineModel.spec.js

```diff
@@ -93,27 +93,6 @@ ava('.getRandomAircraftType() calls ._getRandomAircraftTypeFromFleet() fleet is
     t.true(_getRandomAircraftTypeFromFleetSpy.calledWithExactly(fleetNameMock));
 });
 
-ava('.generateFlightNumber() creates a flightNumber made up of numbers if !flightNumberGeneration.alphaNumeric', (t) => {
```

Comment:
I'd add these two tests back in, but they will fail.  so for now you can do `ava.skip()` and they won't actually affect the rest of the test suite.

these two tests can now be tweaked to work with the new updates.